### DISCUSSION TITLE
Multiple languages in one app

### DISCUSSION THREAD

**Discussion by Hyppias** [SeriousM/WPFLocalizationExtension#348] [Q&A] [answered]
I can show the input screens of my WPF app in 4 languages now, selectable with a ComboBox at runtime.
The calculation output (a Latex text file, which is subsequently compiled into a PDF) I can also show in one of those four languages, however I want to choose the output language independently from the screen language.  Now, when I choose the output language after the calculations are done, the screen language also changes at runtime. I don't want that, of course.
What's the way to go?

**Comment by Karnah** [comment, marked as answer]
Hi, @Hyppias.
It's impossible for LocalizationDictionary to support two cultures simultaneously. But if it's necessary only in one place, you can create and bind your own `CultureInfo` property and change `Content` for your control using `LocalizeDictionary.GetLocalizedObject`

**Reply by Hyppias** [reply]
So I'll try moving the Latex stuff to a class library project in the same solution....
